src/app/pages/HoldRequest.jsx

```diff
@@ -228,12 +228,13 @@ export class HoldRequest extends React.Component {
         : '';
     const itemId = params && params.itemId ? params.itemId : '';
     const selectedItem = bib && itemId ? LibraryItem.getItem(bib, itemId) : {};
-    const selectedItemAvailable = selectedItem
+    const selectedItemAvailable = selectedItem && selectedItem.available
```

Comment:
Optional chaining can be used to abbreviate this a bit: 

const selectedItemAvailable = selectedItem?.available

Reply:
Good point. We're using them elsewhere, so maybe they're getting transpiled? Not really important though since we're upgrading soon.